### PR TITLE
Improve record tab hash handling

### PR DESCRIPTION
**Pull request by rajaro**
Fixes an issue where anchor links can trigger click on links inside elements with the same class name as the new hash

### CODE REVIEW

**Comment by demiankatz** [comment]
Thanks, @rajaro, this makes sense to me. I'm asking @crhallberg and @EreMaijala to double-check in case there are any implications I'm overlooking. I confirmed that the "record-tabs" class is consistently applied in both Record and Collection views, so that potential inconsistency is not an issue. I will also run the full test suite to make sure there are no problems there.

**Comment by demiankatz** [comment]
All tests are passing, which seems like a very good sign... but since there have been some unexpected bugs related to changes to the tab code in the past, I'm going to wait for some additional reviews before merging, out of an abundance of caution.

**Review by EreMaijala — APPROVED** [review]

**Comment by EreMaijala** [comment]
@demiankatz Looks good to me. The old code was indeed a bit scary.

**Comment by demiankatz** [comment]
Thanks, @EreMaijala and @rajaro -- I have merged this now!

**Pull request merged**
